woocommerce/woocommerce-ios
# [Issue Refunds] Add item quantity selection

close #2841 

# Why
This PR takes care of allowing the user to choose how many items they want to refund for every single item.

# How

## On `IssueRefundViewController`
- When the user taps on the quantity button of an item a `ListSelectorViewController` is presented, For this I've created an `ItemListSelectorCommand` class.
- When the user finishes selecting a quantity, the view controller invokes its `ViewModel to handle the new selected quantity.

## On `IssueRefundViewModel`
- Added a new state property `RefundQuantityStore` to store the quantity that a user selects for each item.
- Created methods to get and update the current quantity state for each item
- Updated section creation to reflect the state of `RefundQuantityStore`

# Demo
![select-item-quantity](https://user-images.githubusercontent.com/562080/95520570-34742e00-098d-11eb-8f57-fd69bfdaff93.gif)

# Testing Steps
- Navigate to a not refunded order
- Tap on "Issue Refund Button"
- Play with the item quantity selection and see that values are being reflected correctly.
PS: Summary view with totals and taxes will be done in the next PR



Update release notes:
